tests/contracts/main/application/test_authorization.py

```diff
@@ -0,0 +1,528 @@
+"""
+This file is part of nucypher.
+
+nucypher is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+nucypher is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import pytest
+from eth_tester.exceptions import TransactionFailed
+
+from nucypher.blockchain.eth.constants import NULL_ADDRESS
+from nucypher.blockchain.eth.token import NU
+
+
+AUTHORIZATION_SLOT = 3
+DEAUTHORIZING_SLOT = 6
+END_DEAUTHORIZATION_SLOT = 7
+
+
+def test_authorization_increase(testerchain, threshold_staking, pre_application, token_economics):
+    """
+    Tests for authorization method: authorizationIncreased
+    """
+
+    creator, operator = testerchain.client.accounts[0:2]
+    increase_log = pre_application.events.AuthorizationIncreased.createFilter(fromBlock='latest')
+
+    minimum_authorization = token_economics.minimum_allowed_locked  # TODO
+    value = minimum_authorization
+
+    # Can't call `authorizationIncreased` directly
```

Comment:
Can we clarify _why_ it can't be called?  Or is this so obvious as to not warrant comment?

Given my weak solidity comprehension, my presumption is that it is because `external override onlyStakingContract updateReward(_operator)`, but I'm not entirely sure what that line is doing (which may not be important) or why (which is probably important).